diff --git a/fsw/pc-rtems/src/cfe_psp_start.c b/fsw/pc-rtems/src/cfe_psp_start.c
index 5d03d17..ea26224 100644
--- a/fsw/pc-rtems/src/cfe_psp_start.c
+++ b/fsw/pc-rtems/src/cfe_psp_start.c
@@ -33,8 +33,11 @@
 #include <errno.h>
 #include <rtems.h>
 #include <rtems/rtems_bsdnet.h>
+
+#if 0
 #include <rtems/rtems_dhcp_failsafe.h>
 #include <bsp.h>
+#endif
 
 extern int rtems_fxp_attach(struct rtems_bsdnet_ifconfig *config, int attaching);
 
@@ -62,7 +65,7 @@ extern int rtems_fxp_attach(struct rtems_bsdnet_ifconfig *config, int attaching)
 */
 
 rtems_id RtemsTimerId;
-
+#if 0
 static unsigned char ethernet_address[6] = {0x00, 0x04, 0x9F, 0x00, 0x27, 0x61};
 static char          net_name_str[]      = "fxp1";
 static char          ip_addr_str[]       = "10.0.2.17";
@@ -81,7 +84,7 @@ static struct rtems_bsdnet_ifconfig netdriver_config = {
 struct rtems_bsdnet_config rtems_bsdnet_config = {
     .ifconfig = &netdriver_config, .bootp = rtems_bsdnet_do_dhcp_failsafe, /* fill if DHCP is used*/
 };
-
+#endif
 /*
 ** 1 HZ Timer "ISR"
 */
